src/sage/env.py

```diff
@@ -192,7 +192,7 @@ def var(key: str, *fallbacks: Optional[str], force: bool = False) -> Optional[st
 SAGE_SPKG_INST = var("SAGE_SPKG_INST", join(SAGE_LOCAL, "var", "lib", "sage", "installed"))  # deprecated
 
 # source tree of the Sage distribution
-SAGE_ROOT = var("SAGE_ROOT")  # no fallback for SAGE_ROOT
+SAGE_ROOT = var("SAGE_ROOT", "")  # no fallback for SAGE_ROOT
```

Comment:
```suggestion
SAGE_ROOT = var("SAGE_ROOT")  or None
```
As mentioned, setting this to the empty string will break things